src/types.ts

```diff
@@ -33,6 +35,8 @@ export type AppState = {
   resizingElement: NonDeletedExcalidrawElement | null;
   multiElement: NonDeleted<ExcalidrawLinearElement> | null;
   selectionElement: NonDeletedExcalidrawElement | null;
+  boundElement: NonDeleted<ExcalidrawBindableElement> | null;
```

Comment:
This begs a tsdoc and maybe rename to `boundElementStart` (preferably something better than that). As I understand it, this is solely used when you start creating an arrow by clicking within the binding threshold. If we could keep this in closure without storing to state (I haven't checked if possible), that would be best.